module1/homework/grayscaleImages/compression.cpp

```diff
@@ -0,0 +1,65 @@
+#include "compression.hpp"
+
+#include <iostream>
+
+
+void printMap(const Bitmap& bitmap){
+
+   for(const auto row : bitmap){
```

Comment:
probably this will not print ninja correctly (please look into control symbols (lower that 32) in Ascii table.

Reply:
Could you say something more? I know what a control symbols is, but i don't know why they aren't print correctly.

Reply:
For example backspace or new line are symbols that will break your image. You need to handle them specially. Right now the image is broken.